modules/basegl/glsl/volume_gradient.frag

```diff
@@ -34,15 +34,17 @@ uniform VolumeParameters volumeParameters;
 uniform sampler3D volume;
 
 uniform int channel = 0;
+uniform vec2 dataRange = vec2(0, 1);
 
 in vec4 texCoord_;
 
 void main() {
 #ifdef ADD_DATA_CHANNEL
-    float v = getNormalizedVoxelChannel(volume, volumeParameters, texCoord_.xyz,channel);
+    float v = getNormalizedVoxelChannel(volume, volumeParameters, texCoord_.xyz, channel) 
+        * (dataRange.y - dataRange.x) + dataRange.x;
```

Comment:
This should be value range?

Reply:
No, the intention here is to replicate the original value stored in the input volume without any mapping. For example, `getNormalizedVoxel()` returns a [0,1] range for a uint16 texture with a data range of [0, 65535] independent of ValueRange.